repositories/Repository.js

```diff
@@ -49,14 +49,16 @@ class Repository {
         }
     }
 
+    // ||  Object.keys(data).length === 0
+    // typeof id !== 'string' ||
     async update(id, data) {
         console.log('♥ '+ id);
         try {
-            if (!id || !data || typeof id !== 'string' || Object.keys(data).length === 0)
+            if (!id || !data || typeof id !== 'string' || Object.keys(data).length === 0 )
```

Comment:
it's better to send the params to another function to validate the params.

Reply:
just remind...

Reply:
just remind

Reply:
How did it get here?
It should be in my PR and I fixed it

Reply:
@rachelyWinter maybe it's because this PR is not up to date with the main
so @yehuditNoygershal  pulll the changes and than we'll check it out.